tests/test_jsonschema/test_json_schema_string_format.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from typing_extensions import Annotated
+
+from mashumaro.jsonschema.builder import build_json_schema
+from mashumaro.jsonschema.schema import (
+    JSONSchemaInstanceFormatExtension,
+    JSONSchemaStringFormat,
+)
```

Comment:
Could you import these classes from mashumaro.jsonschema.models directly?

Reply:
Done in [5a292b1](https://github.com/Fatal1ty/mashumaro/pull/232/commits/5a292b13114ff4ccb81d2e37475256d8f0511d11)